src/savegame_json.cpp

```diff
@@ -1513,6 +1514,7 @@ void vehicle::serialize(JsonOut &json) const
     json.member( "camera_on", camera_on );
     json.member( "dome_lights_on", dome_lights_on );
     json.member( "aisle_lights_on", aisle_lights_on );
+    json.member( "has_atomic_lights", has_atomic_lights );
```

Comment:
Indeed, the other things are saved because they can be toggled, but even with this save/load her, the call to refresh later (after all vehicle parts have been loaded) will override the loaded values anyway - sorry, no cheating by editing the saves (-; At least not that easy.